docs/concepts/metrics/available_metrics/context_precision.md

```diff
@@ -13,14 +13,6 @@ $$
 
 Where $K$ is the total number of chunks in `retrieved_contexts` and $v_k \in \{0, 1\}$ is the relevance indicator at rank $k$.
 
-## LLM Based Context Precision
```

Comment:
Why to move this down?

Reply:
Well, to structure the document better. 

Right now, the document has an example that explains the metric of context precision itself (and how its value changes if we add irrelevant context chunks in various positions). We then move to the three subcategories of context precision, each with its own example. 

The example with `LLMContextPrecisionWithoutReference` is repeated twice, since the first time I have used it to illustrate Context Precision as a whole, and the second time is an example of this specific sub-category metric. 

Previewing this document [here](https://github.com/anupamck/ragas/blob/main/docs/concepts/metrics/available_metrics/context_precision.md) and reading it from top to bottom make this change clearer.

Reply:
Makes sense. Will look at it again. Thanks for the explanation and also for raising this. 🙌🏼